examples/nextjs/next.config.js

```diff
@@ -1,6 +1,7 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  swcMinify: true,
```

Comment:
```suggestion
```
This is the default now so we can omit it

Reply:
Awesome will do + I'll make a PR to omit this from the create-next-app package templates as well! Also in regards to merging this PR, I think it would be best to hold off until [v13.0.7](https://github.com/vercel/next.js/releases/tag/v13.0.7-canary.1) is officially released -- lmk what you think!

Reply:
@ijjk re: @styfle's comment below: ah, looks like this template will update itself once 13.0.7 becomes an official release -- so I'm assuming this PR is not needed! 

But I'll still open the PR to omit swcMinify from the create-next-app templates!

cc: @kale-stew